pages/balance-budget.vue

```diff
@@ -204,8 +205,11 @@ export default Vue.extend({
     };
   },
   methods: {
-    initializeTotalAmount() {
-      this.$store.commit('budget/setTotalAmount', TEMP_TOTAL_AMOUNT);
+    initializeTotalAmount(values) {
+      this.$store.commit('budget/setTotalAmount', Object.values(values).reduce((a, b) => a + b));
+    },
+    updateRealAmounts(values) {
```

Comment:
note to self: it might make more sense to have a single `initializeRealAmounts` call that does the total amount computation for you. don't think there's value in having a mutation only for the total amount